h2o-core/src/main/java/water/api/RequestServer.java

```diff
@@ -665,7 +665,7 @@ private static NanoResponse serveError(H2OError error) {
 
   private static NanoResponse redirectToFlow() {
     NanoResponse res = new NanoResponse(HTTP_REDIRECT, MIME_PLAINTEXT, "");
-    res.addHeader("Location", "/flow/index.html");
+    res.addHeader("Location", H2O.ARGS.context_path + "/flow/index.html");
```

Comment:
@micahstubbs @mmalohlava if it's fine with you guys I'd add a JIRA about this (redirect from /) and add it later as Tom mentioned. For now this feature is mainly for the Steam proxy and the redirect won't be necessary. Don't see why usual users would even use the `context_path` argument for now.